src/main/java/tornadofx/LayoutDebugger.kt

```diff
@@ -63,7 +63,7 @@ class LayoutDebugger : Fragment() {
 
     private fun hookListeners() {
         nodeTree.selectionModel.selectedItemProperty().onChange {
-            if (it?.value != null) setSelectedNode(it!!.value.node)
+            if (it?.value != null) setSelectedNode(it.value.node)
```

Comment:
I'm not sure, but maybe use let, like you do on all the other places?

Reply:
If `it` is `null` then the condition will fail and we won't get to the `setSelectedNode` call.